### PR TITLE
perf:lmbench Add parameter for memory value

### PR DESCRIPTION
**Pull request by nasastry**
Add parameter to pass memory value to the lmbench

Signed-off-by: Nageswara R Sastry <rnsastry@linux.vnet.ibm.com>

### CODE REVIEW

File: perf/lmbench.py [hunk 1]
```diff
@@ -101,6 +102,8 @@ def setUp(self):
                        (fsdir, config_file, tmp_config_file))
         process.system("sed 's!^FILE=.*$!FILE=%s!' '%s'  '%s'" %
                        (temp_file, tmp_config_file, config_file))
+        process.system("sed 's!^MB=.*$!MB=%s!' '%s'  '%s'" %
```
Comment (harish-24) [inline perf/lmbench.py]: As this repeats for some params, can we have a small `replace` function to do the same?

Comment (harish-24) [reply perf/lmbench.py]: Better if that can be targeted towards avocado utility too, IMO, would be very useful. @narasimhan-v Any suggestions on this.

Comment (narasimhan-v) [reply perf/lmbench.py]: 1. This is better if done natively in python. Something like below:
```
with open(config_file, "r") as in_file:
    with open(tmp_config_file, "w") as out_file:
        for line in in_file:
            out_file.write(line.replace('old', 'new'))
```

2. Whenever some code is repeating, better to have a loop / function. If that serves multiple tests, then utility. I think local function would do here.

Comment (harish-24) [reply perf/lmbench.py]: @nasastry The first change is from `config_file` to `tmp_config_file`. The other two seems to be opposite of it. Is this right?

Comment (nasastry) [reply perf/lmbench.py]: @harish-24 
`sed` tool changes they are capturing in to a temporary file and then replace original file with the temporary one. This is common for `sed` related operation.
@narasimhan-v and @harish-24 
Let me try a way to replace `sed` commands with `python` code. Will let you know asap.

**Comment by nasastry** [comment]
@narasimhan-v @harish-24 Not understood why it suddenly showing closed and merged. I just sent the updated code. Seems there is a problem. I will create one more PR.

**Pull request closed without merging**